### PR TITLE
Various bug fixes

### PR DESCRIPTION
**Pull request by Wyvest**
## Checklist
- [ ] I made a clear description of what was changed
- [ ] I stated why these changes were necessary
- [ ] I updated documentation or said what needs to be updated
- [ ] I made sure these changes are backwards compatible
- [ ] This pull request is for one feature/bug fix

### CODE REVIEW

**Review by xtrm-en — APPROVED** [review]
lgtm!

File: versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java [hunk 1]
```diff
@@ -110,8 +110,26 @@ public void init() {
...
                 try {
-                    handleForgeCommand(mod);
-                    handleForgeGui(mod);
+                    boolean shouldAttempt = true;
+                    for (Mod configMod : ConfigCore.mods) {
+                        final String configModName = configMod.name.toLowerCase(Locale.ENGLISH);
+                        final String spacelessConfigModName = configModName.replace(" ", "");
+                        if (Objects.equals(configModName, mod.getName().toLowerCase(Locale.ENGLISH)) || Objects.equals(configModName, mod.getModId()) ||
+                                Objects.equals(spacelessConfigModName, mod.getName().toLowerCase(Locale.ENGLISH)) || Objects.equals(spacelessConfigModName, mod.getModId())) {
+                            shouldAttempt = false;
```
Comment (nextdayy) [inline versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java:119]: continue;

File: src/main/java/cc/polyfrost/oneconfig/internal/gui/HudGui.java [hunk 2]
```diff
@@ -100,6 +100,9 @@ public void onDrawScreen(@NotNull UMatrixStack matrixStack, int mouseX, int mous
             if (editingHuds.containsKey(hud)) {
                 nanoVGHelper.setupAndDraw(true, vg -> nanoVGHelper.drawRect(vg, position.getX(), position.getY(), position.getWidth(), position.getHeight(), new Color(0, 128, 128, 60).getRGB()));
```
Comment (nextdayy) [inline src/main/java/cc/polyfrost/oneconfig/internal/gui/HudGui.java:101]: make constant

File: src/main/java/cc/polyfrost/oneconfig/gui/elements/text/TextInputField.java [hunk 3]
```diff
@@ -575,4 +588,17 @@ public void setBoarderThickness(float thickness) {
...
+
+    public static boolean isAnySelected() {
+        return isAnySelected;
+    }
+
+    private static class TextInputSelectionChecker {
+        @Subscribe
+        private void onRender(RenderEvent event) {
+            if (event.stage == Stage.START) {
+                isAnySelected = false;
```
Comment (nextdayy) [inline src/main/java/cc/polyfrost/oneconfig/gui/elements/text/TextInputField.java:600]: this is actually so dumb but okay

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-all.zip
```
Comment (nextdayy) [inline gradle/wrapper/gradle-wrapper.properties:3]: bump with gradlew --gradle-version=8.7

File: versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java (hunk 1 above)
Comment (Wyvest) [reply versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java:119]: Screenshot 2024-04-22 at 10 36 09 AM

**Pull request closed without merging**